### PR TITLE
Convert scripts/nixops to setuptools entry point

### PR DESCRIPTION
**Pull request by adisbladis**
This is more idiomatic & makes it possible to run `python -m nixops`.

### CODE REVIEW

**Comment by teto** [comment]
I love entrypoints as a convenience but this can be costly in startup time https://github.com/papis/papis/issues/217

**Comment by grahamc** [comment]
Mind rebasing (redoing) on current master?

**Comment by adisbladis** [comment]
@grahamc Rebased on master.

File: scripts/nixops [hunk 1]
```diff
@@ -1,262 +1,2 @@
...
-except KeyboardInterrupt:
-    error("interrupted")
-    sys.exit(1)
-except MultipleExceptions as e:
-    error(str(e))
-    if args.debug or args.show_trace or str(e) == "":
-        e.print_all_backtraces()
-    sys.exit(1)
+#!/usr/bin/env bash
+exec python -m nixops "$@"
```
Comment (grahamc) [inline scripts/nixops]: Is this just for backwards compatibility at dev time?

Comment (adisbladis) [reply scripts/nixops]: Exactly.

**Pull request merged**